### PR TITLE
Modify expert bot added

### PR DESCRIPTION
**Pull request by sebix**
* New bot `modify`
* fixes in postgres bot: autocommit, parameter for table
* tests: dictionaries can be given directly, will be converted to json internally
* utility programs (psql init db, harmonization docs generator) are moved to bin/

Harmonization updates from #185
* renamed field `extra` implemented
* `classification.identifier` added

### CODE REVIEW

File: intelmq/__init__.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
-PIPELINE_CONF_FILE = "/opt/intelmq/etc/pipeline.conf"
-STARTUP_CONF_FILE = "/opt/intelmq/etc/startup.conf"
-RUNTIME_CONF_FILE = "/opt/intelmq/etc/runtime.conf"
+DEFAULT_LOGGING_LEVEL = "INFO"
+DEFAULT_LOGGING_PATH = "/opt/intelmq/var/log/"
 DEFAULTS_CONF_FILE = "/opt/intelmq/etc/defaults.conf"
 HARMONIZATION_CONF_FILE = "/opt/intelmq/etc/harmonization.conf"
-DEFAULT_LOGGING_PATH = "/opt/intelmq/var/log/"
-DEFAULT_LOGGING_LEVEL = "INFO"
+MODIFY_CONF_FILE = "/opt/intelmq/etc/modify.conf"
```
Comment (SYNchroACK) [inline intelmq/__init__.py]: I think this modify_config should be in a specific expert folder. The /opt/intelmq/etc is the folder for intelmq "global" configs. Do you agree @sebix ?

Comment (sebix) [inline intelmq/__init__.py]: Where should this file be located after installation? `/opt/intelmq/var/lib/bots/modify/modify.conf`?

Comment (SYNchroACK) [inline intelmq/__init__.py]: yes! :D

**Comment by SYNchroACK** [comment]
great feature!

**Pull request merged**